.github/workflows/site-deployment.yml

```diff
@@ -58,3 +58,10 @@ jobs:
         run: yarn workspace @blockprotocol/site playwright test --project=smoke
         env:
           PLAYWRIGHT_TEST_BASE_URL: ${{ github.event.deployment.environment != 'production' && github.event.deployment_status.target_url || 'https://blockprotocol.org' }}
+
+      - name: Upload artifact playwright-report
+        if: ${{ success() || failure() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: site-smoke-tests-playwright-report
+          path: site/playwright-report
```

Comment:
Setup improvement: this artifact helps debugging failed smoke tests if there are any